### PR TITLE
Add traceID/spanID to structured logs and chunk long logs

### PR DESCRIPTION
**Pull request by squee1945**
@zevdg PTAL

### CODE REVIEW

**Review by joelgerard — APPROVED** [review]

File: internal/log.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	// See http://cloud/logging/quotas#log-limits
+	maxLogMessage = 255000
+)
+
+func logf(c *context, level int64, format string, args ...interface{}) {
+	if c == nil {
+		panic("not an App Engine context")
+	}
+
+	if !IsSecondGen() {
```
Comment (zevdg) [inline internal/log.go:34]: The IsSecondGen() function doesn't make sense in the context of a go112+ only version of this library.  maybe use IsAppengine() or IsStandard() here instead?   IsSecondGen() should probably be removed entirely, but maybe not in this PR.

File: internal/log.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+		s := strings.TrimRight(fmt.Sprintf(format, args...), "\n")
+		now := timeNow().UTC()
+		timestamp := fmt.Sprintf("%d/%02d/%02d %02d:%02d:%02d", now.Year(), now.Month(), now.Day(), now.Hour(), now.Minute(), now.Second())
+		fmt.Fprintf(logStream, timestamp+" "+logLevelName[level]+": "+s+"\n")
```
Comment (zevdg) [inline internal/log.go:38]: nit: fmt.Fprintf(logStream, "%s %s: %s\n", timestamp, logLevelName[level], s)

**Review by zevdg — APPROVED** [review]

File: internal/log.go (hunk 1 above)
Comment (squee1945) [reply internal/log.go:34]: Switched to `IsStandard()`

File: internal/log.go (hunk 2 above)
Comment (squee1945) [reply internal/log.go:38]: Done

**Pull request merged**